lib/rubocop/cop/layout/space_around_keyword.rb

```diff
@@ -30,10 +30,13 @@ class SpaceAroundKeyword < Cop
 
         DO = 'do'
         SAFE_NAVIGATION = '&.'
+        NAMESPACE_OPERATOR = '::'
         ACCEPT_LEFT_PAREN =
           %w[break defined? next not rescue return super yield].freeze
         ACCEPT_LEFT_SQUARE_BRACKET =
           %w[super yield].freeze
+        ACCEPT_NAMESPACE_OP =
```

Comment:
I think `ACCEPT_NAMESPACE_OPERATOR` is clearer than `ACCEPT_NAMESPACE_OP `.

Reply:
Changed to `ACCEPT_NAMESPACE_OPERATOR`.